__init__.py

```diff
@@ -180,21 +185,36 @@ def execute(
         images = images.cpu().numpy() * 255.0
         images = np.clip(images, 0, 255).astype(np.uint8)
 
-        _, format_ext = format.split("/")
+        format_type, format_ext = format.split("/")
 
         tf = tempfile.NamedTemporaryFile()
 
-        frames = [Image.fromarray(f) for f in images]
-        # Use pillow directly to save an animated image to the tmp file
-        frames[0].save(
-            tf.name,
-            format=format_ext.upper(),
-            save_all=True,
-            append_images=frames[1:],
-            duration=round(1000 / frame_rate),
-            loop=0,
-            compress_level=4,
-        )
+        # use pillow for images
+        if format_type == "image":
+            frames = [Image.fromarray(f) for f in images]
+            # Use pillow directly to save an animated image to the tmp file
+            frames[0].save(
+                tf.name,
+                format=format_ext.upper(),
+                save_all=True,
+                append_images=frames[1:],
+                duration=round(1000 / frame_rate),
+                loop=0,
+                compress_level=4,
+            )
+
+        # use ffmpeg for video
+        else:
+            if ffmpeg_path is None:
+                print("no ffmpeg path")
+
+                dimensions='1280x720'
```

Comment:
I would expect `dimensions` to be based on the input images, it's probably fine to base it on the first one.

Reply:
ah, thanks for catching